src/styles/variables.scss

```diff
@@ -0,0 +1,14 @@
+//VARIABLES:
+
+$text-size: 30px;
+$day-size: 100px;
+$calendar-width: $day-size * 7 + 6;
+$days: (
+  "mon": 0,
+  "tue": 101px,
+  "wed": 202px,
+  "thu": 303px,
+  "fri": 404px,
+  "sat": 505px,
+  "sun": 606px
+);
```

Comment:
Strange solution, why hardcode exact size for each case, set day number here and then just calculate needed value